internal/shim-sgx/src/handler/mod.rs

```diff
@@ -71,6 +73,63 @@ impl<'a> Write for Handler<'a> {
     }
 }
 
+trait SecInfoExt {
+    fn accept(&self, dest: NonNull<libc::c_void>);
+    fn accept_copy(&self, dest: NonNull<libc::c_void>, src: NonNull<libc::c_void>);
+    fn protect(&self, dest: NonNull<libc::c_void>);
```

Comment:
Shouldn't they return the error code from rax ??

Reply:
* EACCEPT, EACCEPTCOPY: yes
* EMODPE: no

Reply:
I fixed this issue and pushed a new commit.